tasks/show_summary.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+LOGS=${PT_logs:-/var/log/puppetlabs/puppetserver/puppetserver.log}
+RUBY=$(which ruby||echo "/opt/puppetlabs/puppet/bin/ruby")
+
+echo "Analyzing logs: "
+eval echo $LOGS
```

Comment:
Should avoid `eval`. Example user input: "hello; rm -rf /../"

Reply:
yes, that's why I added the FIXME to actually restrict log paths within /var/log/puppetlabs/puppetserver.

Reply:
Having said that, I'll find a safer way to expand globs.

Reply:
I've removed the eval and replaced by simple echo.